tables/basic-table/step4.md

```diff
@@ -0,0 +1,9 @@
+In this next step, we'll go over what it takes to handle large datasets from a styling perspective.
+
+## Task
+
+1) Set the variant property for the Table component to compact
+
+<pre class="file">
+variant="compact"
```

Comment:
I understand that the variant is being added to the table component, but thinking new users may need a little more help? This is where that "locate the following code" step might help guide them.